lib/global-admin/addon/security/roles/new/route.js

```diff
@@ -13,6 +13,7 @@ export default Route.extend({
       type: `roleTemplate`,
       name: '',
       rules: [],
+      hidden: false,
     });
```

Comment:
Does it make sense? Currently UI can not show the created roles in role list. So I made this change.